### ISSUE TITLE
Communities - Corta el nombre de los users

### ISSUE THREAD

**Issue by IsbetVR** [Fermat-ORG/beta-testing-program#583]
### TKY-ART (Versión v1u4) 

-----

##### Reporte de error: 

   *Al entrar a "Communities" sea de "Artist" o de "Fan", el nombre del usuario lo corta (Pantalla 1, Artist Community), (Pantalla 2, Fan Community).
   
------

#### Pantalla 1

![screenshot_17](https://cloud.githubusercontent.com/assets/19142276/16232447/97f5e6b0-3798-11e6-840c-428e95372760.jpg)

#### Pantalla 2

![screenshot_18](https://cloud.githubusercontent.com/assets/19142276/16232460/a164237e-3798-11e6-94f2-934bc8e8000d.jpg)

------

#### Dispositivo desde donde se realizó la prueba

- [x] Emulador
- Google Nexus 4
- Versión de Android: 5.1.0
- 320 dpi
- API 22 768x1280

![google nexus 4 - 5 1 0](https://cloud.githubusercontent.com/assets/19142276/16232584/1d6bb95a-3799-11e6-9e55-a4097c3348b9.jpg)

**Comment by IsbetVR** [comment]
@MaviDavsa

**Comment by MaviDavsa** [comment]
@darkestpriest

**Comment by darkestpriest** [comment]
Saludos @IsbetVR no entiendo este issue.

**Comment by IsbetVR** [comment]
@darkestpriest saludos! Fíjate por ejemplo en la primera imagen (Pantalla 1) el nombre de "miguel artist" está cortando la letra "g" en "miguel", lo mismo pasa con "Amy", está cortando la letra "y", en la parte de abajo... De igual manera sucede en la segunda imagen (Pantalla 2), el nombre de "Gabby" por ejemplo, corta la letra "y" en la parte de abajo.

**Comment by darkestpriest** [comment]
Cierto, debes perdrnar a este mega miope ;) vamos a invocar a @juanaso para que nos ayude con este fix.

**Comment by lostwood** [comment]
este corrigio para la version 1v10 pueden corroborarlo

**Comment by Hernandezsmanu** [comment]
Solucionado en la v1u11.

**Comment by IsbetVR** [comment]
Solucionado en ambas comunidades (fan y Artist). Versión v1u11

![screenshot_109](https://cloud.githubusercontent.com/assets/19142276/16507029/ab4906a8-3ef4-11e6-947f-1bb4f607cd84.jpg)

### Dispositivo de prueba

- [x] Emulador
- Google Nexus 4
- Versión de Android: 5.1.0
- 320 dpi
- API 22 768x1280
